### ISSUE TITLE
AVT 2 & 3 Radio Button Group and Form Component - Cannot cannot tab to or select radio buttons unselected radio buttons.

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon-components-react#2178]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Tab to Radio button Group - Focus is on default selected radio button
Tab again for Shift-Tab and none of the other radio buttons receive focus.

**Comment by snidersd** [comment]
This issue should remain open until it is fixed and retested.

**Comment by tw15egan** [comment]
Tab focus seems to go to the Radio group, and then the navigation is done via the arrow keys. Is this incorrect?

**Comment by snidersd** [comment]
@t15egan I just retested with VoiceOver and with just the keyboard.  Keyboard navigation is working as expected. Thank you!

**Comment by tw15egan** [comment]
Thanks for letting us know! Going to close this as resolved then
